packages/block-library/src/latest-posts/edit.js

```diff
@@ -470,6 +495,20 @@ export default withSelect( ( select, props ) => {
 	);
 
 	const posts = getEntityRecords( 'postType', 'post', latestPostsQuery );
+
+	let authors = [];
+	if ( posts ) {
+		const postAuthors = posts.reduce( ( accumulator, post ) => {
+			accumulator.push( post.author );
+			return accumulator;
+		}, [] );
```

Comment:
This can be simplified:
```suggestion
		const postAuthors = posts.map( ( post ) => post.author );
```